websocket-lite/examples/hello-world-client.rs

```diff
@@ -43,14 +43,11 @@ async fn run() -> Result<()> {
     Ok(())
 }
 
-fn main() {
-    let rt = tokio::runtime::Runtime::new().unwrap();
-
-    rt.spawn(async {
+#[tokio::main]
+async fn main() {
+    tokio::spawn(async {
         run().await.unwrap_or_else(|e| {
             eprintln!("{}", e);
         })
     });
```

Comment:
Need to `.await.unwrap();` here or the app exits before it does anything.

Reply:
Not sure why it seems that if the [JoinHandle](https://docs.rs/tokio/0.2.2/tokio/task/struct.JoinHandle.html)is dropped the task still executes doesn't it ?

Reply:
The spawned task does continue to execute after JoinHandle is dropped, but the main thread doesn't wait for the spawned task to finish before it exits the process.

Reply:
Ok I understand now will modify